<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(24, 24, 24);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>10</y>
     <width>91</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="layout_Buttons">
    <item>
     <widget class="QPushButton" name="button_Ok">
      <property name="styleSheet">
       <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Cancel">
      <property name="styleSheet">
       <string notr="true">font: 63 10pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 11pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 11pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Type">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 11pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>461</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 11pt &quot;Segoe UI Semibold&quot;;
background-color: rgb(52, 52, 52);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
